src/image-copy/azext_imagecopy/__init__.py

```diff
@@ -48,10 +48,9 @@ def load_arguments(self, _):
                        help='Time in seconds for the copy operation to finish. Increase this time if '
                        'you are going to copy large images (disks) like 512GB or more.')
             c.argument('temporary_resource_group_name',
-                       options_list=['--temporary-resource-group-name', '--temporary_resource_group_name'],
+                       options_list=['--temporary-resource-group-name'],
```

Comment:
Since `--temporary-resource-group-name` is a default option, so we can directly remove it's explicit definition
```suggestion
```